interface/reports/dictation_review.php

```diff
@@ -0,0 +1,316 @@
+
+require_once("../globals.php");
+require_once("$srcdir/patient.inc");
+require_once("$srcdir/formatting.inc.php");
+
+$from_date = fixDate($_POST['form_from_date'], date('Y-m-d'));
+$to_date   = fixDate($_POST['form_to_date'], date('Y-m-d'));
+
+?>
+<html>
+<head>
+    <?php html_header_show();?>
+    <title><?php xl('Dictation Review','e'); ?></title>
+    <?php Header::setupHeader(['datetime-picker', 'report-helper']); ?>
+
+    <script type="text/javascript">
+
+        function submitForm() {
+            var fromDate = $("#form_from_date").val();
+            var toDate = $("#form_to_date").val();
+
+            if (fromDate === '') {
+                alert("<?php echo xls('Please select From date'); ?>");
+                return false;
+            }
+            if (toDate === '') {
+                alert("<?php echo xls('Please select To date'); ?>");
+                return false;
+            }
+            if (Date.parse(fromDate) > Date.parse(toDate)) {
+                alert("<?php echo xls('From date should be less than To date'); ?>");
+                return false;
+            }
+            else {
+                $("#form_refresh").attr("value", "true");
+                $("#report_form").submit();
+            }
+        }
+
+        $( document ).ready(function(){
+            $('.datepicker').datetimepicker({
+                <?php $datetimepicker_timepicker = false; ?>
+                <?php $datetimepicker_showseconds = false; ?>
+                <?php $datetimepicker_formatInput = false; ?>
+                <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
+                <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
+            });
+        });
+
+    </script>
+<style>
+    .cell{
+        display: table-cell;
+        border: 1px solid #d3d3d3;
+    }
+</style>
+</head>
+
+<body class="body_top">
+
+<span class='title'><?php xl('Report','e'); ?> - <?php xl('Dictation Review','e'); ?>    </span>
+<span><?php echo "<br>" . xlt("Click Submit without changing dates to show all dictations"); ?></span>
+<div id="report_parameters_daterange">
+    <?php //echo date("d F Y", strtotime($form_from_date)) ." &nbsp; to &nbsp; ". date("d F Y", strtotime($form_to_date)); ?>
+
+</div>
+
+<form name='theform' method='post' action='dictation_review.php' id='theform' onsubmit='return top.restoreSession()'>
+
+    <div id="report_parameters">
+
+        <input type='hidden' name='form_refresh' id='form_refresh' value=''/>
+        <input type='hidden' name='form_refresh1' id='form_refresh1' value=''/>
+
+        <table>
+            <tr>
+                <td width='410px'>
+                    <div style='float:left'>
+
+                        <table class='text'>
+                            <tr>
+                                <td class='control-label'>
+                                    <?php xl('From','e'); ?>:
```

Comment:
need to remove e parameter and use echo instead and escape via xlt:
``````
<?php echo xlt('From'); ?>: